PyInstaller/utils/hooks/__init__.py

```diff
@@ -41,11 +41,11 @@
 hook_variables = {}
 
 
-def __exec_python_cmd(cmd, env=None):
+def __exec_python_cmd(cmd, env=None, ret_stdout=True):
```

Comment:
I suggest naming this `capture_stdout", like the resp. option in `subprocess` is called.

Reply:
Good idea.